employee-rostering-frontend/spring-boot/java/org/optaweb/employeerostering/frontend/WebConfig.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaweb.employeerostering.frontend;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+
+@Configuration
+public class WebConfig extends WebMvcConfigurerAdapter {
```

Comment:
Why mixing backend Java code to the frontend project? I think this should go to the backend.

Reply:
I choose to put this code here because it is specific to the react frontend. Different frontend frameworks may use different conventions for URL's, and may have multiple html files. This code is not needed for the backend, only for connecting the backend to the frontend.

Reply:
Btw, the adapter is deprecated.

Reply:
You're right that from the backend perspective it's specific to the React fronted. At the same time it is true that from the frontend perspective, it's specific to the case where the frontend is server by the Spring Boot app. If I serve the frontend with NGNIX this piece of Spring Boot configuration is irrelevant.

Could you move it to the module where frontend and backend are packaged together?

Reply:
+1 Great idea!